crm_phonecall/models/crm_phonecall.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2004-2016 Odoo SA (<http://www.odoo.com>)
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from functools import reduce
+
+from odoo import api, fields, models
+
+
+class CrmPhonecall(models.Model):
+    """Model for CRM phonecalls."""
+
+    _name = "crm.phonecall"
+    _description = "Phonecall"
+    _order = "id desc"
+    _inherit = ["mail.thread", "utm.mixin"]
+
+    date_action_last = fields.Datetime(string="Last Action", readonly=True)
+    date_action_next = fields.Datetime(string="Next Action", readonly=True)
+    create_date = fields.Datetime(string="Creation Date", readonly=True)
```

Comment:
```suggestion
    date_action_last = fields.Datetime(string="Last Action")
    date_action_next = fields.Datetime(string="Next Action")
    create_date = fields.Datetime(string="Creation Date")
```
`readonly=True` makes impossible to import these fields. OCA migration guidelines invite us to consider to set the attribute at view level.